vue/src/vueTextMask.js

```diff
@@ -1,24 +1,42 @@
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
```

Comment:
I prefer we don't add the `.js` extension to imports in this project. At this point, this is simply for consistency with the rest of the codebase.